chicken-category.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Noodles with chicken</title>
+  </head>
+  <body>
+    <h1>Noodles with chicken</h1>
+    <ul>
+      <li>
+        <a href="./chicken-pad-thai.html"><h3>Chicken Pad Thai</h3></a>
```

Comment:
usually we would nest the `a` tag inside the `h3` tag (e.g. `<h3><a>Chicken Pad Thai</a></h3>`